packages/workspace/src/generators/init/init.spec.ts

```diff
@@ -56,6 +56,10 @@ describe('workspace', () => {
         '/tsconfig.spec.json',
         '{"extends": "../tsconfig.json", "compilerOptions": {}}'
       );
+      tree.write(
```

Comment:
This looks completely separate? Can updates to `init.ts` be in a separate PR?

Reply:
It's not separate at all. The `init` generator is broken after the change introduced in this PR. It's broken for a couple of reasons:

- This generator uses `readJson` to read the configuration files instead of the `readProjectConfiguration` or similar utilities that normalize the result to the v2 format.
- Because of the above and because the update configuration methods were always writing v2 format, there were pieces of code using `architect` and others using `targets` based on when the update happened.

So, given that updating the configuration will now always write in the right format. I needed to either keep using the `readJson` and change everything to use `architect` or start using the devkit utilities and use `targets`. I did the latter to ensure consistency.